Changelog.md

```diff
@@ -1,4 +1,4 @@
-### 0.4.18 (unreleased)
+### 0.4.18 (2017-10-18)
 
 Features:
  * Code Generator: Always use all available gas for calls as experimental 0.5.0 feature
```

Comment:
Can all these be sorted?

Also capitalise `Tangerine Whistle`.

Reply:
I actually think we should sort by decreasing importance, although that is often hard to judge.

Reply:
I think all them were alphabetically sorted except the last one. I think that's good, but the summary on the top usually gives the subjective opinion on importance.

Actually, that summary on the top (which is in the release notes at the github tag) should be also included in the real file as it has useful information.